index.html

```diff
@@ -4,11 +4,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="style.css" rel="stylesheet">
+    <script src="https://use.fontawesome.com/23bea62652.js"></script>
     <title></title>
 </head>
 <body>
     <div class="main-container" >
-         
-    </div>
+         <section class="contact-section">
```

Comment:
`div class="main-container"`  must be inside section

Reply:
done